src/pages/events/create.svelte

```diff
@@ -0,0 +1,162 @@
+<script>
```

Comment:
This is my fault, but let's rename this to new.svelte to be consistent with the `requests/new` route.